lib/cucumber/glue/snippet.rb

```diff
@@ -88,7 +88,7 @@ def typed_pattern
         def to_s
           header = generated_expressions.each_with_index.map do |expr, i|
             prefix = i == 0 ? '' : '# '
-            "#{prefix}#{code_keyword}(\"#{expr.source}\") do#{parameters(expr)}"
+            "#{prefix}#{code_keyword}('#{expr.source}') do#{parameters(expr)}"
```

Comment:
Have you considered the case where `expr.source` contains a single quote?

Reply:
Or, if we keep the double-quoted strings, the case where `expr.source` contains a `\` character.

Reply:
Ooh! Good point. I actually agree with everything you’re saying and would also prefer the double quotes to stay. I just misread your other comments and took them to mean you preferred this way, and deferred to that even though I didn’t agree 😂

Reply:
I’m making myself a note to submit new documentation changes tomorrow morning and change this PR right away.